### ISSUE TITLE
How can I add Mean Average Precision metric?

### ISSUE THREAD

**Issue by ftcister** [tensorflow/recommenders#696]
Hello, I was reading several books on recommender systems and in the literature the **mAP (Mean Average Precision)** metric is used a lot and I wanted to know how I can implement it, in the tutorials and in the API only TopK Accuracy appears, I was wondering if anyone has already implemented mAP@K, thanks.

**Comment by rlcauvin** [comment]
It looks like you may use [tfr.keras.metrics.MeanAveragePrecisionMetric](https://www.tensorflow.org/ranking/api_docs/python/tfr/keras/metrics/MeanAveragePrecisionMetric).
